src/internet_identity/internet_identity.did

```diff
@@ -215,6 +215,17 @@ type IdentityAnchorInfo = record {
     device_registration: opt DeviceRegistrationInfo;
 };
 
+type AnchorCredentials = record {
+    credentials : vec WebauthnCredential;
+    recovery_credentials : vec WebauthnCredential;
+    recovery_phrase: bool;
```

Comment:
really nit: since we send the pubkeys everywhere, does it make sense to send it for the recovery phrase as well?

Reply:
I made it a `Vec<PubKey>` because there might still be anchors with multiple recovery phrases (from before we enforced the limit).